smarts/core/sensors.py

```diff
@@ -123,6 +130,33 @@ class Vias:
     """List of points that were hit in the previous step"""
 
 
+class TrafficLightState(Enum):
+    UNKNOWN = 0
+
+    # States for traffic signals with arrows.
+    ARROW_STOP = 1
+    ARROW_CAUTION = 2
+    ARROW_GO = 3
+
+    # Standard round traffic signals.
+    STOP = 4
+    CAUTION = 5
+    GO = 6
+
+    # Flashing light signals.
+    FLASHING_STOP = 7
+    FLASHING_CAUTION = 8
+
+
+@dataclass(frozen=True)
+class TrafficLightData:
+    stop_point: Tuple[float, float]
```

Comment:
We could use `Point` here (from `smarts.core.coordinates`).  ,,,
But is it really just a point and not a line segment?  If so, this seems to imply that a traffic light can only control a single lane.  I still feel like we must be missing something about the waymo dataset if this is the case!

Reply:
Thanks for the tip, I'll use `Point` instead.

For your other comment -- the traffic light data is given on a per-lane basis rather than per-light, so each point represents the "stop point" in a single lane if the relevant light is in a "stop" state. There is no concept of a "light" on its own. This is how the Waymo dataset defines it, we are simply passing it on.

Here's an image from a scenario to illustrate:
![image](https://user-images.githubusercontent.com/17256344/132764088-078fd95f-aeb9-4df7-8c37-cca7da58e459.png)